tools/mo/openvino/tools/mo/ops/loop.py

```diff
@@ -536,6 +536,7 @@ def normalize_input_output_ports(loop_node: Node):
 
     @staticmethod
     def infer(loop_node: Node):
+        Loop.normalize_input_output_ports(loop_node)
```

Comment:
Why we need it here?

Reply:
We don't need it here, it was redundant call.